test/integration/examples/apiserver_test.go

```diff
@@ -357,13 +359,32 @@ func TestAggregatedAPIServer(t *testing.T) {
 	// (the service is missing), we don't have an external signal.
 	time.Sleep(100 * time.Millisecond)
 	_, err = aggregatorDiscoveryClient.Discovery().ServerResources()
-	if err != nil {
-		t.Fatal(err)
-	}
+	assertWardleUnavailableDiscoveryError(t, err)
 
 	// TODO figure out how to turn on enough of services and dns to run more
 }
 
+func assertWardleUnavailableDiscoveryError(t *testing.T, err error) {
+	if err == nil {
```

Comment:
call `t.Helper()` to make sure failures report the line number where this was called